test-cases/longevity/longevity-parallel-schema-changes-12h.yaml

```diff
@@ -20,7 +20,7 @@ n_monitor_nodes: 1
 instance_type_db: 'i4i.2xlarge'
 
 nemesis_class_name: 'SisyphusMonkey:1 SisyphusMonkey:1'
-nemesis_selector: [["topology_changes"],["!disruptive","schema_changes"]]
+nemesis_selector: [["topology_changes"],["not disruptive","schema_changes"]]
```

Comment:
Shouldn't second list stop having 2 values?
And if yes, do we need inner type as list after implementing this feature?

Reply:
change all of them, and switch to use a string instead of a list